BitmovinConvivaAnalytics/Classes/Helper/BitmovinPlayerHelper.swift

```diff
@@ -30,6 +30,6 @@ final class BitmovinPlayerHelper: NSObject {
     }
 
     var version: String? {
-        return Bundle(for: PlayerFactory.self).infoDictionary?["CFBundleShortVersionString"] as? String
+        Bundle(for: PlayerFactory.self).infoDictionary?["CFBundleShortVersionString"] as? String
```

Comment:
nit: we should use the official API for getting the version
```suggestion
        PlayerFactory.sdkVersion
```

Reply:
Actually this could fix some tracking if falling into the same issue as we've seen before with overridden versions in `Info.plist` files

Reply:
[dbb962b](https://github.com/bitmovin/bitmovin-player-ios-analytics-conviva/pull/65/commits/dbb962b0b8e4ce29fa1f1c88b5184ca955387bc3)